Dockerfile

```diff
@@ -0,0 +1,30 @@
+ARG work_dir=/go/src/github.com/Bnei-Baruch/galaxy-monitor
+
+FROM golang:1.14-alpine3.11 as build
+
+LABEL maintainer="bbfsdev@gmail.com"
+
+ARG work_dir
+
+ENV GOOS=linux \
+	CGO_ENABLED=0 \
+
+RUN apk update && \
+    apk add --no-cache \
+    git
+
+WORKDIR ${work_dir}
+COPY . .
+
+RUN go test $(go list ./... | grep -v /models) \
```

Comment:
redundant grep. you don't have models folder

Reply:
Done.